src/components/gltf-model-plus.js

```diff
@@ -340,6 +340,8 @@ AFRAME.registerComponent("gltf-model-plus", {
       if (gltf.animations.length > 0) {
         this.el.setAttribute("animation-mixer", {});
         this.el.components["animation-mixer"].initMixer(gltf.animations);
+      } else {
+        this.el.removeAttribute("animation-mixer");
```

Comment:
Probably makes sense to always do this cleanup in case the animation-mixer has any other state. Putting it in `removeInflatedEl` (maybe with a rename) seems like a good candidate, since that also gets called in the empty src case above